packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -348,11 +348,13 @@ describe('CheckoutModule', () => {
         let commit;
         let dispatch;
         let payload;
+        let rootGetters;
 
         beforeEach(() => {
             commit = jest.fn();
             dispatch = jest.fn();
             state = defaultState;
+            rootGetters = {};
```

Comment:
To make the action invocations look a bit neater, these could be wrapped up into a `context` object, e.g.:

```suggestion
        let commit;
        let dispatch;
        let payload;
        let rootGetters;
        let context;

        beforeEach(() => {
            commit = jest.fn();
            dispatch = jest.fn();
            state = defaultState;
            rootGetters = {};
            context = { commit, state, dispatch, rootGetters };
```